ccmbioinfo/lfs
# LFS-630: Add an extension point allowing the insertion of visual element at the top of each page

This PR implements LFS-630 and allows for a visual element to be added at the top of each page. There are several use cases for this feature including LFS-622 which adds a _demo mode_ warning banner to the top of each page.

To test:
- Build the `LFS-630_test` branch.
- Start LFS normally with `java -jar distribution/target/lfs-0.1-SNAPSHOT.jar`.
- Everything should work as usual.
- Restart LFS in the `demo` run mode. `rm -rf sling/` , `java -jar distribution/target/lfs-0.1-SNAPSHOT.jar -Dsling.run.modes=demo`
- An _orange_ demo warning should be displayed at the top of each page.
- Restart LFS in the `hello` run mode. `rm -rf sling/` , `java -jar distribution/target/lfs-0.1-SNAPSHOT.jar -Dsling.run.modes=hello` 
- A _green_ hello world banner should be displayed at the top of each page.
